com.unity.ml-agents/Runtime/Agent.deprecated.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using UnityEngine;
+
+namespace Unity.MLAgents
+{
+    public partial class Agent
+    {
+        // [Obsolete("CollectDiscreteActionMasks has been deprecated. Please use WriteDiscreteActionMask instead.", false)]
+        public virtual void CollectDiscreteActionMasks(DiscreteActionMasker actionMasker)
+        {
+        }
+
+        // [Obsolete("The Heuristic(float[]) method has been deprecated.  Please use Heuristic(float[], int[]) instead.")]
+        public virtual void Heuristic(float[] continuousActionsOut)
```

Comment:
Thinking about this a bit more, maybe we should keep the Heuristic behavior the same until we actually have a way to specify continuous and discrete actions in BrainParameters. Even if you use `Heuristic(float[], int[])` you can't actually use both outputs yet.

If you still want to go ahead with it, I think you should keep the old argument name since these aren't necessarily continuous.

Reply:
Just saw the name change.  Was a mistake.  I'll change it back to `actionsOut`.  

We can keep this method around as long as we like and not obsolete it.  Does that work for you?

Reply:
Sounds good!